examples/multimap/event-listener/main.go

```diff
@@ -0,0 +1,103 @@
+package main
+
+import (
+	"context"
+	"log"
+
+	hz "github.com/hazelcast/hazelcast-go-client"
+	"github.com/hazelcast/hazelcast-go-client/types"
+)
+
+func main() {
+	ctx := context.TODO()
+
+	client, err := hz.StartNewClient(ctx)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	m, err := client.GetMultiMap(ctx, "my-multimap")
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	myHandler := func(event *hz.EntryNotified) {
+		switch event.EventType {
+		case hz.EntryAdded:
+			log.Printf("Multimap: %s, (key: %v, value: %v) was added.\n", event.MapName, event.Key, event.Value)
+		case hz.EntryRemoved:
+			log.Printf("Multimap: %s, (key: %v, value: %v) was removed.\n", event.MapName, event.Key, event.OldValue)
+		case hz.EntryAllCleared:
+			log.Printf("Multimap: %s was cleared.\n", event.MapName)
+		}
+	}
+
+	myAwesomeKey := "my-awesome-key"
+
+	listenerConfigs := [...]hz.MultiMapEntryListenerConfig{
```

Comment:
Instead of creating an array of listener config, can we define them close to their usage? Since there are only two, I feel like it would be simpler without an array.